lib/cli/commands/walletwithdraw.ts

```diff
@@ -3,16 +3,16 @@ import { WithdrawRequest } from '../../proto/xudrpc_pb';
 import { callback, loadXudClient } from '../command';
 import { coinsToSats } from '../utils';
 
-export const command = 'walletwithdraw [amount] [currency] [destination] [fee]';
+export const command = 'walletwithdraw <amount> <currency> <destination> [fee]';
 
 export const describe = 'withdraws on-chain funds from xud';
 
 export const builder = (argv: Argv) => argv
-  .option('amount', {
-    description: 'the amount to withdraw',
-    type: 'number',
+  .positional('amount', {
+    description: 'the amount to withdraw, `all` withdraws everything',
+    type: 'string',
   })
-  .option('currency', {
+  .positional('currency', {
     description: 'the ticker symbol of the currency to withdraw.',
     type: 'string',
   })
```

Comment:
I noticed one more thing here on another pass, the line below with `option('destination'` should be `positional('destination'`. Sorry if that's a headache with the commits, but you should be able to make the change and just do:

```
git add -A
git commit --amend --no-edit
git push --force
```

And it should update this PR with just that one line below.

Reply:
I guess it worked, thanks a ton @sangaman :)